### PR TITLE
Fix for Code in comments overflowing

### PR DESCRIPTION
**Pull request by jm90m**
* https://github.com/busyorg/busy/issues/771

### CODE REVIEW

**Comment by jm90m** [comment]
Example here: `/philosophy/@michellecarter/what-is-the-highest-power` in the comments

**Comment by Sekhmet** [comment]
I've tried this solution but it causes scrollbars not showing up which is not ideal for code blocks. Code is used to be displayed without braking lines with scrollbars if needed.

**Comment by jm90m** [comment]
@Sekhmet @bonustrack Updated styles on this, should look like this now w/ scrollbar:

example here: `/test/@hellosteem/5ihr1p-test#@hellosteem/re-hellosteem-re-hellosteem-re-hellosteem-5ihr1p-test-20171018t142248652z`

screen shot 2017-10-18 at 5 24 23 pm
screen shot 2017-10-18 at 5 24 28 pm

**Comment by bonustrack** [comment]
The element is bigger than expected, i'm not able to see the right part of the scrollbar
image

File: src/components/Story/Body.less [hunk 1]
```diff
@@ -99,7 +99,8 @@
   }
 
   pre {
-    overflow-y: auto;
+    overflow: auto;
+    max-width: 90%;
```
Comment (bonustrack) [inline src/components/Story/Body.less]: There should be a better way than this :/

**Comment by jm90m** [comment]
@bonustrack check latest commit, that should fix it, removed max-width

**Comment by jm90m** [comment]
thank you 🙏  @bonustrack

**Pull request merged**